src/js/blue-button/utils/helpers.js

```diff
@@ -0,0 +1,34 @@
+import environment from '../../common/helpers/environment';
```

Comment:
This was copy pasted from rx, but probably worth consolidating this into a common helper at some point (also present in messaging and disability benefits)